brianlong/www.validators.app
# 178998318_add_average_to_batches_controller_changes

#### What's this PR do?
- Change way of gathering data for average

#### How should this be manually tested?
- Part 2/2, first merge and run script from 178998318_reduce_n_plus_1_queries_on_v_block_history_show #179
- Open show page for any validator
- check Skipped Slot History table (best way to check it is to run first master, see the table, then switch to this branch and check if the results has not been changed)

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178998318
- 
#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
